#X-Generator: crowdin.com
# Content-related messages

content.content_missing=D\u00fc\u011f\u00fcm i\u00e7eri\u011fi eksik\: \n   d\u00fc\u011f\u00fcm\: {0} \n   okuyucu\: {1}  \n   L\u00fctfen sistem y\u00f6neticiniz ile g\u00f6r\u00fc\u015f\u00fcn.
content.runtime_exec.property_moved=The property 'errorCodes' has moved down onto the RuntimeExec class

index.recovery.out_of_date=Dizinler veritaban\u0131 ile e\u015fle\u015ftirilmemi\u015f.
index.tracking.starting=Dizinler yeniden olu\u015fturuluyor.
index.tracking.complete=Dizinler yeniden olu\u015fturuldu.
index.tracking.progress=\t{0} etraf\u0131ndaki hareketler i\u015fleniyor.
index.recovery.starting=Dizin yeniden olu\u015fturuluyor\: {0} hareket.
index.recovery.complete=Dizinler yeniden olu\u015fturuldu.
index.recovery.progress=\ % {0} tamamland\u0131.
index.recovery.terminated=Dizin olu\u015fturma sonland\u0131r\u0131ld\u0131.

node.archive.msg.busy=Toplu bir silme ya da yenileme i\u015flemi s\u00fcr\u00fcyor

content.http_reader.err.no_connection=HTTP ile Alfresco sunucusuna ba\u011flan\u0131lamad\u0131\: {0}
content.http_reader.err.no_authentication=HTTP okuyucu uzak sunucuda oturum a\u00e7amad\u0131\: {0}  \n
content.http_reader.err.check_cluster='replicateUpdates' ve 'replicateUpdatesViaCopy' ayarlar\u0131n\u0131n 'org.alfresco.cache.ticketsCache' \u00f6nbelle\u011fi i\u00e7in etkinle\u015ftirildi\u011finden emin olun. Genel k\u00fcme ayarlar\u0131n\u0131n do\u011fru ve \u00e7al\u0131\u015f\u0131yor oldu\u011fundan emin olun.
content.http_reader.err.unrecognized=Uzak sunucudan i\u00e7erik indirilmeye \u00e7al\u0131\u015f\u0131l\u0131rken bir hata olu\u015ftu\:\n   Sunucu\: {0} \n   \u0130\u00e7erik\: {1} \n   HTTP Yan\u0131t\u0131\: {2}

metadata.extraction.err.type_conversion=\u00dcstveri de\u011feri istenen tipe d\u00f6n\u00fc\u015ft\u00fcr\u00fclemedi\u011fi i\u00e7in ay\u0131klanamad\u0131\: \n   Ay\u0131klay\u0131c\u0131\: {0} \n   Hedef \u00d6zelli\u011fi QName\: {1} \n   \u0130stenen Tip\: {2} \n   Ay\u0131klanan De\u011fer\: {3}

transform.err.format_or_password=Hatal\u0131 bi\u00e7imde ya da parola ile korunmu\u015f bir dosya nedeniyle i\u00e7erik d\u00f6n\u00fc\u015ft\u00fcr\u00fclemedi.

content.routing.err.invalid_default_store=''defaultStoreName'', ''{0}'', ''storesByName'' ({1}) i\u00e7indeki bir ma\u011fazaya kar\u015f\u0131l\u0131k gelmiyor.
